src/ExtMongoDbResourceManager.php

```diff
@@ -66,7 +65,7 @@ public function setResource($id, $resource)
                 '%s expects an array or %s; received %s',
                 __METHOD__,
                 Collection::class,
-                is_object($resource) ? get_class($resource) : gettype($resource)
+                is_object($resource) ? $resource::class : gettype($resource)
```

Comment:
TIL, all of this can be replaced with `get_debug_type($resource)`

Reply:
I wonder if this could be handled by [phpcbf](https://github.com/squizlabs/PHP_CodeSniffer/blob/master/bin/phpcbf) or rector.

I personally would prefer not to make these changes manually.

/cc @samsonasik

Reply:
I saw it today in a @samsonasik rector run on laminas-filter 💪

Reply:
@boesing @gsteel yes, there is rector rule for that https://github.com/rectorphp/rector-src/blob/main/build/target-repository/docs/rector_rules_overview.md#getdebugtyperector